components/list/test/list-item-checkbox-mixin.test.js

```diff
@@ -32,10 +32,10 @@ describe('ListItemCheckboxMixin', () => {
 	describe('Does not render checkbox or action area when not selectable', () => {
 		const cases = [{
 			input: '',
-			expected: { selectable: undefined, disabled: undefined, selected: undefined }
+			expected: { selectable: undefined, disabled: undefined, selected: false }
```

Comment:
I'm not totally sure why these didn't fail when I added the initial state assignment in my last change.  It must have something to do with the timing.  Regardless, the expected value is `false` now.